### PR TITLE
Extractor Grant Application

### PR DESCRIPTION
**Pull request by andriyk-hacken**
# Grant Application Checklist

- [ ] The [`applications/application-template.md`](../applications/application-template.md) or [`applications/research-template.md`](../applications/research-template.md) has been copied, renamed ( "project_name.md") and updated.
- [ ] An Ethereum address for the payment of the milestones is provided inside the application.  
- [ ] The total funding amount of the project is up to USD $30k for every grant applications.
- [ ] The initial PR contains only one commit (squash if needed before submitting your PR).

### CODE REVIEW

File: applications/extractor.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+<img src="https://github.com/haas-labs/extractor/raw/main/doc/Ext-Screen-3.png" width="800">
+
+<br>
+<img src="https://github.com/haas-labs/extractor/raw/main/doc/Ext-Screen-4.png" width="800">
+
+<br>
+
+__Brief Product Slidedeck__:
+
+[https://github.com/haas-labs/extractor/blob/main/doc/Extractor-VeChain.pdf](Extractor-VeChain.pdf)
```
Comment (libotony) [inline applications/extractor.md:43]:
```suggestion
[Extractor-VeChain.pdf](https://github.com/haas-labs/extractor/blob/main/doc/Extractor-VeChain.pdf)
```

File: applications/extractor.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+- Andriy K (https://www.linkedin.com/in/andriy-kashcheyev-a019382/)
+- Andriy U (https://www.linkedin.com/in/andreyusov/)
+- Gleb K (https://www.linkedin.com/in/gleb-kozyryatskyy-51074395/)
+- Vova G (https://www.linkedin.com/in/vlgutv/)
+
+
+## Development Roadmap 
+
+#### Overview
+
```
Comment (libotony) [inline applications/extractor.md:99]: Our policy stipulates that the maximum allocated budget is $30,000. Please review and adjust your budget accordingly. Thank you for your cooperation.

**Review by libotony — CHANGES_REQUESTED** [review]
Please take a look at the comments above.

File: applications/extractor.md (hunk 2 above)
Comment (andriyk-hacken) [reply applications/extractor.md:99]: Will have to sync with colleagues on this one.

**Comment by andriyk-hacken** [comment]
Updated accroding to comments

**Comment by libotony** [comment]
@andriyk-hacken Looks like you have deleted milestone 4, so this won't be a full integration for vechain? Do you have plans for milestone 4?

**Comment by andriyk-hacken** [comment]
> @andriyk-hacken Looks like you have deleted milestone 4, so this won't be a full integration for vechain? Do you have plans for milestone 4?

The core VeChain integration is in Milestone-1 and 2. Other milestones target Product advanaced functionality to bring VeChain on-par with our current EVM support capabilities. With proposed budget cap it is impossible to implement Milestone-4. Milestone-3 will have some limitations too.

**Comment by NecoSherry** [comment]
Congratulations! Your application has been accepted by the grant committee.

Before making the payment of M1, we kindly request you to confirm if your address can receive USDC (ERC-20).
Additionally, we will communicate payment details with you via email. Please **DO NOT** leave your email in the application. Instead, send an email to [grant@vechain.org](mailto:grant@vechain.org) including your project name, PR link, and the payment address (which should match the one provided in your application).

**Review by NecoSherry — APPROVED** [review]

**Review by libotony — APPROVED** [review]

**Comment by andriyk-hacken** [comment]
Hello @NecoSherry,
Sorry for the prolonged silence. 
Some time ago we have deployed VeChain on Production with features of Milestone-2 and Milestone-3 implemented. We are going to publish updates on our media about new chain support and use case article.
What steps are needed from us to complete the grant process ?

**Pull request merged**